### PR TITLE
Global summit 2015

### PR DESCRIPTION
**Pull request by latinogan**
### Capstone Project
- added html file
- added css file
- added linters to the project
- Create the voice of costumer dynamically in js

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    </header>
+    <section>
+        <div class="main">
+            <h1>"Hello, Sharing world!"</h1>
+            <p class="creative">Travelers Rental cars.<br> 2020</p>
+            <div class="joyfull">
+                <p>A joyfull celebration believing in the value of opennes and sharing. creating a positive change with people from over 80 countrys is taking place in October. in USA</p>
+                <p class="dates">2022.04.14(THU)</p>
+                <p class="national">@ National  museum of USA. Art Center</p>
+            </div>
```
Comment (vic778) [inline index.html:60]: please the `font` should match the design of the template. remember that this project is to help you perform your skills in front of pages

File: about.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            <h4  class="active" >Travelers Rental <br>Car</h4>
+
+            <div id="Links">   
+                <ul>
+             <li ><img src="./pictures/close.png" class="close" alt="Close menu"></li>  
+             <li>  <a href="about.html" class="test">About</a></li> 
+             <li>  <a href="#" class="test">Program</a></li> 
+             <li>  <a href="#" class="test">Join</a></li> 
+             <li>  <a href="#" class="test">Sponsor</a></li> 
+             <li>  <a href="index.html" class="rent">Rent a Car</a></li>
```
Comment (vic778) [inline about.html:47]: kindly try to fix the border-radius of this `Rent a car` and make it bold

File: script.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    city: '.Illinoise',
+    experience: 'treavelers car rentals was a great chiose good costumer service and also they are aware of the costumer was a great expirience ',
+  },
+  {
+    img: './pictures/client6.png" class=person alt="client1',
+    name: 'Abraham chantre.',
+    city: '.Ohio',
+    experience: 'treavelers car rentals was a great chiose good costumer service and also they are aware of the costumer was a great expirience ',
+  },
+];
```
Comment (vic778) [inline script.js]: - kindly this section should contain a square background for each image.
- according to the [template] the section should be centered.  please add a margin for this section.

**Review by vic778 — CHANGES_REQUESTED** [review]
## Hello @latinogan 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
-  Good video description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (latinogan) [reply index.html:60]: change font got it going to change it thank you very much for your CR

File: about.html (hunk 2 above)
Comment (latinogan) [reply about.html:47]: got it Thank you very much

File: script.js (hunk 3 above)
Comment (latinogan) [reply script.js]: a squere going to check and change it thank you coach

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @latinogan, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**